app/src/test/java/com/example/benchproject/app/movie/details/viewModel/MovieDetailsViewModelTest.kt

```diff
@@ -0,0 +1,66 @@
+package com.example.benchproject.app.movie.details.viewModel
+
+import com.example.benchproject.app.movie.details.entity.Genres
+import com.example.benchproject.app.movie.details.entity.MovieDetails
+import com.example.benchproject.domain.movie.details.entity.GenresModel
+import com.example.benchproject.domain.movie.details.entity.MovieDetailsModel
+import com.example.benchproject.domain.movie.details.usecase.GetMovieDetailsUseCase
+import com.example.benchproject.test.common.LiveDataTest
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.amshove.kluent.any
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.extension.ExtendWith
+
+@OptIn(ExperimentalCoroutinesApi::class)
+@ExtendWith(LiveDataTest::class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Why it's added?

Reply:
To be honest, I used it earlier for some reason since I had a problem and now I checked it and it is not needed here like in another test file `PopularMoviesViewModelTest`